app/services/backend/private-cloud/products.ts

```diff
@@ -147,6 +147,13 @@ export async function getPodUsageMetrics(licencePlate: string, environment: stri
   return response.data;
 }
 
+export async function getSubnetForEmerald(licencePlate: string, environment: string, cluster: string) {
+  const response = await instance.get(
```

Comment:
let's set the type of the response data like `instance.get<string>`.

Reply:
@junminahn the response type has been set.